blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClient.scala

```diff
@@ -12,33 +12,31 @@ import scalaz.{-\/, \/-}
 object BlazeClient {
   def apply(manager: ConnectionManager, idleTimeout: Duration, requestTimeout: Duration): Client = {
     Client(Service.lift { req =>
-      def tryClient(client: BlazeClientStage, freshClient: Boolean, flushPrelude: Boolean): Task[DisposableResponse] = {
```

Comment:
Without being able to force a fresh connection there is no way to distinguish between a server that disconnects without sending any bytes for whatever reason and an infinite pool of stale connections. In the former situation this could result in an infinite loop.

I don't know how reasonable a concern that is.